SSO-Auth/Config/configPage.html

```diff
@@ -1,15 +1,272 @@
+                  is="emby-input"
+                  id="OidClientId"
+                  required=""
+                  type="text"
+                />
+                <div class="fieldDescription">The OpenID client ID.</div>
+              </div>
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="OidSecret"
+                  >OID Secret:</label
+                >
+                <input is="emby-input" id="OidSecret" required="" type="text" />
+                <div class="fieldDescription">
+                  The OpenID Secret. Randomly Generated & Shared.
+                </div>
+              </div>
+
+              <div class="checkboxContainer checkboxContainer-withDescription">
+                <label>
+                  <input id="Enabled" name="Enabled" type="checkbox" />
+                  <span>Enabled</span>
+                </label>
+                <div class="fieldDescription checkboxFieldDescription">
+                  Enable the provider
+                </div>
+              </div>
+
+              <div class="checkboxContainer checkboxContainer-withDescription">
+                <label>
+                  <input
+                    id="EnableAuthorization"
+                    name="EnableAuthorization"
+                    type="checkbox"
+                  />
+                  <span>Enable Authorization by Plugin</span>
+                </label>
+                <div class="fieldDescription checkboxFieldDescription">
+                  Determines if the plugin sets permissions for the user. If
+                  false, the user will start with no permissions and an
+                  administrator will add permissions. The permissions of
+                  existing users will not be rewritten on subsequent logins.
+                </div>
+              </div>
+
+              <div class="checkboxContainer checkboxContainer-withDescription">
+                <label>
+                  <input
+                    id="EnableAllFolders"
+                    name="EnableAllFolders"
+                    type="checkbox"
+                  />
+                  <span>Enable All Folders</span>
+                </label>
+                <div class="fieldDescription checkboxFieldDescription">
+                  Determines if the user logging in is allowed access to all
+                  folders.
+                </div>
+              </div>
+
+              <!-- Make this multiline, one per line and then split on newline -->
+
+              <div class="inputContainer">
+                <label
+                  class="inputLabel inputLabelUnfocused"
+                  for="EnabledFolders"
+                  >Enabled Folders:</label
+                >
+                <input is="emby-input" id="EnabledFolders" type="text" />
+                <div class="fieldDescription">
+                  JSON Array of Strings. If <strong>enableAllFolders</strong> is
```

Comment:
Instead of asking for JSON, maybe use what the LDAP plugin uses? https://github.com/jellyfin/jellyfin-plugin-ldapauth/blob/476156b8fae962e67daa92079e4e0c5273335455/LDAP-Auth/Config/configPage.html#L132

Reply:
Shelved for now (WIll be revisted)